hedera-node/hedera-app/src/main/module-info.java

```diff
@@ -0,0 +1,3 @@
+module com.hedera.hashgraph.app {
+
```

Comment:
If we mandate this library to depend on the `com.hedera.hashgraph.app.api` module at the Gradle level as compile & runtime scoped dependency, then we should mirror the requirement at the JPMS module level. 

```suggestion
requires com.hedera.hashgraph.app.api;
```